File: DAY-ONE/JAVA/NumberApp.java
import java.util.Scanner;

import java.util.random;

public class NumberApp{

    public static void main(String[] args){

Scanner myscanner = new Scanner(System.in);


for (int count = 0; count < 5; count++){
System.out.println("Guess number ");
int number = myscanner.nextInt();

if (number <= 100){
System.out.println("Correct Number");break;
}

else if (number > 100){
System.out.println("number is higher ");
}
   }
  }
}

File: DAY-ONE/PYTHON/NumberApp.py
import random

Random = 56
for random_number in range(1,5):

    guess = int(input("Guess number "))
    
    
    if (guess > Random):
        print("Number is lower")
    
    elif (guess < Random):
        print("Number is higher")

    if (guess == Random):
        print("Number is correct ")
        break

    elif(guess > 100):
        print("warning number is not accepted")

    print(random.randrange (1, 100))

    while True:
        correct_guess = 5

    if (correct_guess == 1):
        print("Lengendary ")

    elif (correct_guess == 2):
        print("Excellent ")

    elif (correct_guess == 3 or 4):
        print("Good ")

    else:
        print("No win Better luck")

print("The coerrect guessing number 56 ")
